presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadataFactory.java

```diff
@@ -38,6 +38,7 @@
     private final boolean bucketWritingEnabled;
     private final boolean skipDeletionForAlter;
     private final boolean writesToNonManagedTablesEnabled;
+    private final boolean createNonManagedTableEnabled;
```

Comment:
You're using `createNonManagedTableEnabled` and `createNonManagedTablesEnabled`. I think it would be nice to keep is consistent - `createNonManagedTablesEnabled` like we have for others  - `WritesToNonManagedTablesEnabled`